superset/assets/cypress_build.sh

```diff
@@ -9,8 +9,7 @@ superset/bin/superset runserver &
 
 cd "$(dirname "$0")"
 
-npm install -g yarn
-yarn
+yarn install --frozen-lockfile
 npm run build
-npm run cypress run --record
+npm run cypress run
```

Comment:
Is there a reason you removed `--record`? It doesn't work atm but I'm trying to get it to work, unless we think we don't want to use cypress dashboard.

Reply:
Was trying to make it work via `npm run cypress run -- --record` then cypress complains asking for `--key`, which is required for recording. Will leave this for @michellethomas to handle in future review.